packages/headless/src/features/facets/range-facets/generic/range-facet-controller-actions.ts

```diff
@@ -42,9 +36,26 @@ export const executeToggleRangeFacetSelect = createAsyncThunk<
   }
 );
 
-const getSelectionDefinition = (selection: RangeFacetValue) => {
-  if (typeof selection.start === 'string') {
-    return new RecordValue({values: dateFacetValueDefinition});
+/**
+ * Executes a search with the appropriate analytics for a range facet breadcrumb
+ * @param payload (RangeFacetSelectionPayload) Object specifying the target facet and selection.
+ */
+export const executeRangeFacetBreadcrumb = createAsyncThunk<
```

Comment:
Would `deselectRangeFacetBreadcrumb` be a clearer name?

Reply:
It would be select IMO, for a breadcrumb, but deselect for a facet value, no?